src/pages/extending-chat-widget/rich-messages/index.mdx

```diff
@@ -116,7 +116,7 @@ Let's go through each element:
         * `phone` - allows Customers to call the number given directly from the chat.
     * `elements.buttons.postback_id` - a description of the sent action.
     * `elements.buttons.user_ids` - an array of the users who clicked the button. It can be empty.
-    * `elements.buttons.value` - defines what value the button should have depending on its type. It's either a message to be sent, a URL to be opened in a Moment or in a new tab, or a phone number to call.
+    * `elements.buttons.value` - defines what value the button should have depending on its type. It's either a URL to be opened in a Moment or in a new tab, or a phone number to call. For `message` type it is not a message that would be sent after clicking, in this case the message is aways equal to the `label` property.
```

Comment:
```suggestion
    * `elements.buttons.value` - defines what value the button should have, depending on its type. It's either a URL to be opened in a Moment or in a new tab, or a phone number to call. For `message` type it is not a message that would be sent after clicking, in this case the message is aways equal to the `label` property.
```

Reply:
Im not sure here, commas work a bit differently in english than they do in polish 🤔 We are not changing the context here by any means so therefore I don't think the comma is needed, wdyt @kamilatokarska ?

Reply:
In general, both ways are okay. However, I would leave it without the comma here because we consider `depending on its type` essential to understanding the sentence.